db/migrations/20200919113958_create_urns_by_ilk_query.sql

```diff
@@ -0,0 +1,27 @@
+-- +goose Up
+-- Function returning state for all urns from a given ilk as of given block
+CREATE FUNCTION api.get_urns_by_ilk(ilk_identifier TEXT, block_height bigint DEFAULT api.max_block(),
+                                    max_results integer DEFAULT NULL::integer,
+                                    result_offset integer DEFAULT 0) RETURNS SETOF api.urn_snapshot
+    LANGUAGE sql
+    STABLE
+AS
+$$
+SELECT *
+FROM (SELECT DISTINCT ON (urn_identifier, urn_snapshot.ilk_identifier) urn_identifier,
+                                                          urn_snapshot.ilk_identifier,
```

Comment:
Not a big deal, but my editor formatted this a little differently lining up line 12 indent more to start under `urn_identifier`. I know we always struggle with keeping the styling on `sql` files uniform. My settings might be bunk.